platform/osx/app/MainMenu.xib

```diff
@@ -473,10 +473,16 @@
                                     <action selector="dropManyPins:" target="-1" id="Rtv-8N-3Z8"/>
                                 </connections>
                             </menuItem>
-                            <menuItem title="Remove All Pins" id="6rC-68-vk0">
+                            <menuItem title="Draw Example Polygon and Polyline" id="DVr-vT-lpe">
```

Comment:
Should be called “Add Polygon and Polyline”, since it isn’t an interactive drawing tool.